kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -269,63 +269,76 @@ ip4configProperties, new AccessPointsProperties(activeAccessPoint, accessPoints)
     }
 
     public synchronized void apply(Map<String, Object> networkConfiguration) throws DBusException {
-        doApply(networkConfiguration);
-        this.cachedConfiguration = networkConfiguration;
+        try {
+            configurationEnforcementDisable();
+            doApply(networkConfiguration);
```

Comment:
You need to clear the modem reset timers here.

```suggestion
            configurationEnforcementDisable();
            modemResetHandlersDisable();
            doApply(networkConfiguration);
```

Reply:
Ahhh... maybe I miss it during the rebase.